magenta-lib/src/main/scala/magenta/deployment_type/SelfDeploy.scala

```diff
@@ -53,26 +53,16 @@ object SelfDeploy extends DeploymentType {
       )
     )
   }
+
+
   val selfDeploy = Action("selfDeploy",
     """
-      |Switches the `shutdown-when-inactive` switch to true on the target hosts (discovered using Prism to lookup
-      |hosts according to the stack, app and stage of the deployment).
-      |
-      |The switch is expected to be a [guardian management](https://github.com/guardian/guardian-management) switch.
+      |Invokes the Shutdown controller on the target instance.
     """.stripMargin){ (pkg, resources, target) =>
     implicit val keyRing = resources.assembleKeyring(target, pkg)
-    val reporter = resources.reporter
+
     val hosts = resources.lookup.hosts.get(pkg, pkg.app, target.parameters, target.stack).toList
-    hosts.map{ host =>
-      ChangeSwitch(
-        host,
-        managementProtocol(pkg, target, reporter),
-        managementPort(pkg, target, reporter),
-        switchboardPath(pkg, target, reporter),
-        "shutdown-when-inactive",
-        desiredState=true
-      )
-    }
+    hosts.map(ShutdownTask.apply)
```

Comment:
Guardian Management automatically presented endpoints to toggle switches on a non-standard port. This has been replaced with a custom endpoint (on the standard 9000) to do this.

Reply:
I guess this allows us to remove [this ingress rule](https://github.com/guardian/deploy-tools-platform/blob/ea015f04e297291ad7b2066095459c273b8e447e/cloudformation/riffraff/riffraff.template.yaml#L141-L146)?